builder/context.go

```diff
@@ -61,8 +61,8 @@ func (b *Builder) getDockerRunArgs(
 	if step.User != "" {
 		sb.WriteString(" --user " + step.User)
 	}
-	if step.Network != "" {
-		sb.WriteString(" --network " + step.Network)
+	for _, network := range step.Networks {
```

Comment:
Does `docker run` allow multiple networks? How does it work?

Reply:
No, but it's possible to connect multiple. I'm actually going to revert this change for now for simplicity, we can decide later if we want to introduce it